debian/changelog

```diff
@@ -1,3 +1,21 @@
+opx-platform-config (1.6.0) unstable; urgency=medium
+
+  * Update: Added media configuration files
+  * Update: Added Dell EMC S5248F-ON
+  * Update: Added bcm_env.cfg with 32M dmasize for S5248-ON
+  * Update: Updated copyrights
```

Comment:
You can remove stuff about updating copyrights.

Reply:
done